usaspending_api/awards/v2/data_layer/orm.py

```diff
@@ -356,10 +356,16 @@ def _fetch_parent_award_details(parent_award_ids: dict) -> Optional[OrderedDict]
         else None
     )
 
+    agency_id = parent_agency["id"] if parent_agency else None
+    agency_name = parent_agency["toptier_agency__name"] if parent_agency else None
+
+    has_agency_page = agency_has_file_c_submission(agency_id)
```

Comment:
`agency_has_file_c_submission` uses the `toptier_agency_id` and this use appears to be passing in the `id` from the Agency table.